client/package.json

```diff
@@ -36,6 +36,7 @@
     "ember-cli-eslint": "3.0.0",
     "ember-cli-flash": "1.3.17",
     "ember-cli-focus-input": "1.0.0",
+    "ember-cli-google-analytics": "^1.5.0",
```

Comment:
Since NPM doesn't have a lock file, we've been locking things to exact versions throughout this file. We should figure out the latest version `^1.5.0` resolves to, change this to that version, and remove the `^`.